src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -301,6 +303,70 @@ ggplot(temperature_archery, aes(x = DT_round, y = p1)) +
   theme_bw()
 ```
 
+```{r}
+# Determine each site and parameter in all_data 
+sites <- unique(all_data$site)
+params <- unique(all_data$parameter)
+# Constructing a df to iterate over each site-parameter combination
+combinations <- crossing(sites, params)
+
+# Function to summarize site-parameter combinations
+summarize_site_param <- function(site_arg, parameter_arg, api_data) {
```

Comment:
praise: this is an excellent use of functions/prints/etc. 

suggestion: because this applies to all parameters across all sites, move this up to a section before we look at any individual site/parameters. Its current location doesn't make sense, IMO.